.github/workflows/lint.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v4
+        with:
+          go-mod-file: 'go.mod'
+          cache: false
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
```

Comment:
Probably a silly question, but are we sure this fails CI when imports are incorrect?

Reply:
fair point, I'll do a sanity check tomorrow.

Reply:
@benhoyt did a test [here](https://github.com/barrettj12/pebble/actions/runs/5469046259/jobs/9957477703). Linting indeed fails, but it's not very descriptive - it doesn't even say which files need fixing. I'll see if I can improve this output.

Reply:
I've improved the action output on failure - [here](https://github.com/barrettj12/pebble/actions/runs/5469442058/jobs/9958385473) is a sample run.